### PR TITLE
Update Crossplay to v1.3

### PR DESCRIPTION
**Pull request by Moneylover3246**

### CODE REVIEW

**Review by hakusaro — CHANGES_REQUESTED** [review]
Can you update your assembly version to 1.3.1?

**Comment by Moneylover3246** [comment]
The recent commit was most likely the last breaking bug. Everything should be ok now 🎆

**Review by hakusaro — CHANGES_REQUESTED** [review]
Please version your plugins (do not just simply provide links to versions that work with given terraria versions), so that if we have to tell people to update your plugin, they can do so easily

**Comment by Marotheit** [comment]
You also misspelled "Download". Figured you'd like to know.

File: README.md [hunk 1]
```diff
@@ -153,8 +153,9 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
 * [Crossplay](https://github.com/Moneylover3246/Crossplay) by Moneylover3246
   * Allows crossplay to happen in terraria servers via packet handling and manipulation
   * Tested on TShock 4.5.4.
-  * Extremely experimental, use at your own risk!
-  * [Download Version 1.2.1](https://files.catbox.moe/cvjtow.dll)
+  * Please contact Moneylover3246#0322 on discord if there are any crashing issues!
+  * [Download v1.3.3 - Desktop](https://files.catbox.moe/1p1j8r.dll)
```
Comment (hakusaro) [inline README.md:157]: 1p1j8r ok

File: README.md [hunk 2]
```diff
@@ -153,8 +153,9 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
...
+  * [Download v1.0.0 - Mobile](https://files.catbox.moe/x4sib0.dll)
```
Comment (hakusaro) [inline README.md:158]: x4sib0 ok-ish

_cue nothing else matters song_

**Review by hakusaro — APPROVED** [review]

**Pull request merged**